example.env

```diff
@@ -29,12 +29,14 @@ AWS_API_KEY=
 VPC_ID=
 
 # id of series to recieve ingests in the absence of a zoom meeting -> series mapping
-# this is meant for testing/dev only
+# this is meant for testing/dev only. If you set this to a valid Opencast series id
+# then any recording that doesn't match something in the class schedule data will
+# be ingested into that default series id.
 DEFAULT_SERIES_ID=
```

Comment:
Had a comment on this that leaving "DEFAULT_SERIES_ID=" in the .env file will result in the default series being set to "None" and then the recording being ingested into opencast with an empty series name but then I realized this should be fixed in the uploader.